civis/response.py

```diff
@@ -190,3 +195,8 @@ def __iter__(self):
                 yield converted_data
 
             self._params['page_num'] += 1
+
+    def __next__(self):
+        if self._iter is None:
+            self._iter = self._get_iter()
+        return next(self._iter)
```

Comment:
Does this need a `next` method to be Python 2 compliant?

https://stackoverflow.com/questions/29578469/how-to-make-an-object-both-a-python2-and-python3-iterator